src/drive/mobile/styles/uploadprogression.styl

```diff
@@ -33,9 +33,11 @@
         position absolute
         top 50%
         left 50%
-        max-width 3rem
-        max-height 3rem
         transform translateX(-50%) translateY(-50%)
+        width 3rem
+        height 3rem
+        object-fit cover
```

Comment:
TIL. But are you sure about this change? It looks like `cover` might [crop the image](https://developer.mozilla.org/fr/docs/Web/CSS/object-fit)?

Reply:
That's the point. It's just a preview to know which image we're talking about but that's pretty much it and any image that is not a square will be cropped.

Reply:
Oooh sorry, of course. Somehow in my mind this was a fix for the viewer in photo so I was a bit concerned. My bad!